src/service/standard-actions-impl.js

```diff
@@ -68,6 +68,17 @@ export class StandardActions {
     /** @const @private {!./viewport/viewport-impl.Viewport} */
     this.viewport_ = Services.viewportForDoc(ampdoc);
 
+    // A meta[name="amp-action-whitelist"] tag, if present, contains, 
+    // in its content attribute, a whitelist of actions on the special AMP target. 
+    const meta =
+        window.document.head.querySelector('meta[name="amp-action-whitelist"]');
```

Comment:
`this.ampdoc.getRootNode().head`, to support multi-doc mode. I had to do the same for XHR interception:

https://github.com/ampproject/amphtml/blob/578f7080b595c858c41329713181663b489a1f43/src/service/xhr-impl.js#L190

Reply:
done